sites/en/intro-to-rails/ruby_language.step

```diff
@@ -15,7 +15,7 @@ steps do
 
     message "Yours might look different, but it should look something like this:"
 
-    console_without_message "1.9.3p362 :001 > "
+    console_without_message "irb(main):001:0>"
```

Comment:
My IRB actually still more resembles the other way, insofar as it has numbers in it and everything
```
traviss-air:railsbridge tjgrathwell$ irb
2.2.0 :001 >
```

but really as long as it ends in an angle bracket it's probably fine